### DISCUSSION TITLE
Resellers of Chia-Plots want $15/plot, 16x/day is $240, ROI on Hpool is $1/day

### DISCUSSION THREAD

**Discussion by room101-dev** [Chia-Network/chia-blockchain#5754] [General]
What I really wanted to know is how much is amazon aws charging suckers to make plots?

***

People can you see where this is going? If your good wokesters and solo-mine you get zero forever.
If you run hpool, you get some pocket change.

If your a fool and buy into the 'farming' gig on your mobile phone, and say you bought 16 plots for $240, you could earn a whopping $1/day from HPOOL, and a huge ZERO from chia-net's way of doing things.

Anybody want to guess who is going to make money here? Who is going to lose?

**[ My hpool return is based on the fact that at today's 8EB space, 400 plots on hpool earns you 1 xch monthly ]**
...

I did find this company https://plottair.com/ so this is a good start, they want 12 BP ( british pound), what's that 15 USD? So I could be selling my 16x plots for 16*15 = 240$ /day

Let's see, thats 0.0012 xch day, or $1.03/day of chia income

So I can mine on hpool with 16 plots and get $1.03, or solo-mine with chia-net and get zero, or I could get $240/day selling my plots

You guess where this is all going? Right now my plots ( 108gb ) sit on my HDD, and they don't go anywhere, the mining-pool just sends a few bytes home to confirm I have a valid plot every few minutes, low bandwidth. Can you imagine when every body starts uploading 'plots' to to where? Where the hell is this storage, and for how much??

I can only imagine it must be file-coin?? Other than AWS, or google, who else has massive  storage? The folks above say you plot can be downloaded 3 times for 30 days, so they have a cap on bandwidth & storage.

I have always felt that the scam of chia was to target millenials wokesters, this is all making sense so they can do all this on their mobile phones and be green and never have to touch real hardware.

Lastly, the same guy who sponsors the chia-calculator ( bullshit in my opinion ), is the one selling plots and selling  chia. IMHO the only real barometer on how much a payout on a plot is, is how much hpool pays you for a plot and its right now about 1XCH/MONTH for 400 plots.

***

I dabble in chia-plotting do about 16x day plot generation, for the last few months. A plot is large 108gb file created similar to a bitcoin sha-256 hash, the name of the file represents the hash of that 'plot' (file); The space of chia is all the plots combined at that point in time given  8% daily growth that is 51k new plots added to the network daily

Morons solo mine, what they don't understand is that last month (1eb) they needed 1k plots to win, this month you need 10k plots to win. This is why I pool-mine using hpool, which has been around for months.

So, I have a good idea of what I earn, right now at current space EB is 7.4, e.g. 7.4 billion-billion bytes

400 plots can earn 1 chia every month on hpool (dot com) right now, so 1 plot earns 7 cents / day.

I'm curious how much it costs to host and generate plots, AWS ain't cheap the resources to make a plot are enormous 6-8 hours and 1.5 trillion random r/w on a virtual-disk

Just curious if there is anybody out there stupid enough to do this, and what your paying.

Lot's of people are now offering 'plots' for sale; but I never see the price, given how much it costs to make plots ( I mean I burn up one nvme on every 100 plot creation ), and you need a super-computer, and 3 nvme system, ... its not a cheap hobby to make plots.

So lots of people who want to be green think "I will just buy plots, or farm them on amazon on my mobile phone", ... how much is this costing you? For 7 cents a day, that drops by 1/2 every nine days ( chia space grows 8% per day ), doubles every 9 days, ergo your income on fixed plots 1/2 every 9 days.

**Comment by Team4N6** [comment]
Go home, you're drunk?

You said a lot. I think I get the general idea of what you're saying, but the simple answer is that chia is young and the pooling protocol isn't done yet. It needs to be released for things to normalize. Like every other crypto, ROI is completely dependent on the value of the crypto. Who knows where that will land. I agree though that the massive size of the network has made the idea of a solo farmer untenable.

Places selling plots for $10 is a scam directed at uninformed people, or people that are really bad at math.

**Reply by room101-dev** [reply]
> Go home, you're drunk?
> 
> You said a lot. I think I get the general idea of what you're saying, but the simple answer is that chia is young and the pooling protocol isn't done yet. It needs to be released for things to normalize. Like every other crypto, ROI is completely dependent on the value of the crypto. Who knows where that will land. I agree though that the massive size of the network has made the idea of a solo farmer untenable.
> 
> Places selling plots for $10 is a scam directed at uninformed people, or people that are really bad at math.

Well first  if all its 6am where I live, and no I'm not drunk, in fact last night I wasn't drunk either. R U retarded, They said $15 for a plot, where's your $10 from?

CHIA ain't young the incompetent dev team has been working on this pile of bugs since 2017, that's now going on almost 5 years of low quality sw dev. The pooling protocol will never be finished by the team, they don't even have a working blockchain that can sync. Did U even bother to notice there are 1100 reported outstanding bugs, and nothing ever gets fixed here?

ROI is what it is, and  you clearly don't have an iota of knowledge on this subject. I have explained all, I actually do this shit, rather than insult people about alcohol abuse, I'm going to have assume that your living the streets of PDX, that your a meth user, and a Wokester, and you believe in all chia mythology? Right? That chia is going to lift your life out of the gutter.

My point is/was "Is anybody running AWS to create chia plots, and farm them?" U didn't even address that question, which tells me the OP triggered you on your gov give mobile phone, and you responded on that basis.

***

Lastly, everything about CHIA is a scam, the DEV, the company, the GREEN-BS, the influencers, and the mythology. Chia was designed to create a new market to sell crypto, to 'woke millenials', with the green BS; It's clearly working, as 10's of 1,000's of people here are plotting earning zero, all the while the whales are making Millions of dollars a day mining on HPOOL.

**Reply by Team4N6** [reply]
Ouch. No offense was intended. Sorry about that.

Also, just to be clear, Chia is not a meth heads game. It's too slow and requires patience. Clearly, much more of a heroin addicts crypto. Also, in PDX "living in the gutters" actually means posh shelters. My decked out cot and portable hard drive game are making all the meth heads jealous.
